file-formats/sush/README.md

```diff
@@ -0,0 +1,5 @@
+# SUSH File Format
+
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.sush.json` | 1 | [`zif_aff_sush_v1.intf.abap`](./type/zif_aff_sush_v1.intf.abap) | [`sush-v1.json`](./sush-v1.json) | [`z_aff_example_sush.sush.json`](./examples/z_aff_example_sush.sush.json)
```

Comment:
The example file is missing. We either remove the link and create an issue or (even better) you upload also the example.